### DISCUSSION TITLE
How to use the infisical secret in your kubernetes app after adding it

### DISCUSSION THREAD

**Discussion by davidshare** [Infisical/infisical#2359] [General]
I have self-hosted infisical on my kubernetes cluster, created a secret, and added a project. However, I do not see how to use that secret in the docs.

Is there something I am missing?

For example, how do I attach the secrets I have added in my project to my kubernetes pod?

**Comment by maidul98** [comment]
You will need to user the kubernetes operator https://infisical.com/docs/integrations/platforms/kubernetes

**Reply by davidshare** [reply]
Thank you. So after installing the chart, I still need to install the operator?
